src/label-syncer.ts

```
@@ -0,0 +1,152 @@
+	}
+
+	private extractLinkedIssues(body: string): number[] {
+		if (!body?.trim()) return []
+
+		const closingKeywords = ['close', 'closes', 'fix', 'fixes', 'fixed', 'resolve', 'resolves', 'resolved']
+		const regex = new RegExp(`(?:${closingKeywords.join('|')})\\s+#(\\d+)`, 'gi')
+		const matches = [...body.matchAll(regex)]
+		return matches
+			.map((match) => {
+				const num = parseInt(match[1], 10)
+				return isNaN(num) || num < 1 ? null : num
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)